client/extensions/woocommerce/app/order/order-payment/table.js

```diff
@@ -43,16 +43,17 @@ class OrderRefundTable extends Component {
 
 	constructor( props ) {
 		super( props );
-		const shippingTax = getOrderShippingTax( props.order );
-		const shippingTotal = parseFloat( shippingTax ) + parseFloat( props.order.shipping_total );
+		const { order } = props;
+		const shippingTax = getOrderShippingTax( order );
+		const shippingTotal = parseFloat( shippingTax ) + parseFloat( order.shipping_total );
```

Comment:
I was just noticing that this constructor builds state based on props, but what happens if `props.order` is changed after the component is mounted? I think the props and the state would be out of sync.